google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +188,17 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def update_schema(table):
+        """Maintain all the schema changes for this table.
+
+        Args:
+            table (Table): The table object of this class.
+        """
+        col = Column('resource_name', String(256), default='')
+        LOGGER.info('Attempting to create column: %s', col.name)
+        col.create(table, populate_default=True)
```

Comment:
if the pattern is to add the column as you mentioned, then should we surround try/except around each `col.create()`?  Otherwise, when adding `xyz` in the future, would the execution reach `xyz` if `resource_name` already exists?
```
         col = Column('resource_name', String(256), default='')
         col.create(table, populate_default=True)

         col = Column('xyz', String(256), default='')
         col.create(table, populate_default=True)
```